exec/exec.go

```diff
@@ -29,13 +29,23 @@ var cl = channel.NewLocalChannel()
 
 // stop hang process
 func Destroy(ctx context.Context, c spec.Channel, action string) *spec.Response {
-	ctx = context.WithValue(ctx, channel.ProcessKey, action)
+	suid := ctx.Value(spec.Uid)
```

Comment:
Hello, @Super-long , maybe this uid can't be nil? Because there is code  in ```main.go```
```
		uid := expModel.ActionFlags[model.UidFlag.Name]
		if uid == "" {
			uid, _ = util.GenerateUid()
		}
```